src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -208,12 +208,7 @@ public IAfterCallConfiguredWithOutAndRefParametersConfiguration<IReturnValueConf
             {
                 Guard.AgainstNull(valueProducer, nameof(valueProducer));
                 this.ParentConfiguration.AddRuleIfNeeded();
-                this.ParentConfiguration.RuleBeingBuilt.UseApplicator(call =>
-                {
-                    TMember returnValue;
-                    returnValue = valueProducer(call);
-                    call.SetReturnValue(returnValue);
-                });
+                this.ParentConfiguration.RuleBeingBuilt.UseApplicator(call => call.SetReturnValue(valueProducer(call)));

```

Comment:
Like this, or is it too much?

Reply:
I'm content! The extra line with the uninitialized variable just bugged me. And probably would introduce a nullability warning! :grin:
I didn't care whether you initialized right away or elided the variable.

Reply:
I was too slow to tell you I was okay with amend. Squash it and I'll merge!